### ISSUE TITLE
 report a bug no date issued next to the link approving the signature

### ISSUE THREAD

**Issue by Alexis-M51** [LibreSign/libresign#487]
Hello,

I report a bug no date issued next to the link approving the signature:

version: libresign 2.4.5, installation without docker:

procedure:
-  no date associated with the signature at the bottom of the signed document

**Comment by vitormattos** [comment]
Can you explain more?
You want see on footer of document the date when the document was signed?

**Comment by Alexis-M51** [comment]
yes that's it, because at the bottom there is validated ... and there is no date

**Comment by vitormattos** [comment]
This isn't a bug, this is a feature request.

Is possible that many people sign the document, on this case what will be the date that we will put on footer?

As a soon we will launch a new release to make possible that put visible signatures on document. I think that will be a good feature make possible to when request to sign a document define if want to put a field with timestamp above the signature image, but, this is ideas.

For now, to look the date and time that the document was signed,  you can look on the validation page or look on your pdf reader locally looking all signatures of the PDF.
